corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -804,6 +808,16 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             $.getScript(gridPolyfillPath);
 
             registerContinueListener(this, options);
+            this.handleSmallScreenChange(this.smallScreenEnabled);
+        },
+
+        handleSmallScreenChange: function (enabled) {
+            CaseTileListView.__super__.handleSmallScreenChange.apply(this, arguments);
```

Comment:
I'm not completely understanding this. Won't `this.smallScreenEnabled` be indefinitely set as `enabled` after initialization? What's the purpose of `handleSmallScreenChange` in the parent class?

Reply:
`handleSmallScreenChange` is run as the callback function when the screen size passes the small screen threshold -- that's happening in the `CaseListView`, where `this.smallScreenEnabled` being updated is the only thing that happens in the callback. This adds functionality to the same callback function (so, `this.smallScreenEnabled` still gets updated correctly) without duplicating it.

Reply:
Oh i see, in `self.handleSmallScreenChange.bind(self)`, self refers to either `CaseTileListView` or `CaseListView` as needed and their respective `handleSmallScreenChange` is set as the callback

Reply:
Yep, since the `CaseTileListView` inherits that functionality unless overwritten

Reply:
Got it, thanks for the explanation. The js syntax tripped me up a bit